lib/core/src/sdk.rs

```diff
@@ -1917,7 +1896,13 @@ impl LiquidSdk {
                     },
                     is_confirmed: is_tx_confirmed,
                 },
-                None,
+                match tx.outputs.first() {
+                    Some(Some(output)) => Some(PaymentDetails::Liquid {
+                        destination: output.script_pubkey.to_hex(),
+                        description: "Liquid transfer".to_string(),
```

Comment:
What do you think about leaving the description empty? I see that we fill it with default description when constructing the payment from the db which is better and more flexible than storing.